ptf/saimpls.py

```diff
@@ -3240,11 +3240,8 @@ def mplsEgressPhpTtlModeTest(self):
             eth_src=ROUTER_MAC,
             ipv6_dst='300::1',
             ipv6_hlim=63)
-        # Note: mpls_tag ttl is copied from top label, but it is not
-        #       being decremented even though it should in uniform mode.
-        # Would we really pop labels but not swap what is left?
-        # Maybe this should change to swap with explicit null label?
-        mpls_tag['ttl'] = 56
+        
+        mpls_tag['ttl'] = 55
```

Comment:
Although 55 is correct from a standards point of view, that is not what we implement. This check should be removed.

Reply:
I opened SWI-4748 for this issue.